testing/test_aux.out

```diff
@@ -21,20 +24,121 @@ pri.flows
 pri.ofctl
 sec.flows
 sec.ofctl
-{
-  "brute": {
-    "enabled": true
-  },
-  "foobar": {
-    "enabled": false
-  },
-  "nmap": {
-    "enabled": true
-  },
-  "switch": {
-    "enabled": true
-  }
-}
 Redacted docs diff
-No report diff
+1c1
```

Comment:
I can see this is a diff of some sort.
No idea what the comparison is and why different data is appearing

Reply:
Could be because you're doing it from the wrong file. one is 03, other is 01 -- that shouldn't change.